src/blocks/battery.rs

```diff
@@ -183,6 +186,18 @@ impl BatteryDevice for PowerSupplyDevice {
         }
     }
 
+    fn capacity_level(&self) -> Result<String> {
```

Comment:
Let's move this and the logic from lines 944-961 to the `capacity` function to avoid unnecessary impls for other backends.

Reply:
Would you prefer displaying the Strings from capacity_level or rather display them converted to some percentages? (5/25/50/75/100)?

Reply:
> display them converted to some percentages?

I think that's what Android does, and this make sense for `percentage` placeholder

Reply:
Okay makes it easier to implement as  well.